joomla/joomla-cms
# fix problem with accordions using parent option

Pull Request for Issue # .
### Summary of Changes

Now `accordion-toggle` of slides which where collapsed by opening an other slide in an accordion with the option parent set to the id of the accordion gets the class `collapsed`.
### Testing Instructions

For making the testing simple I've created a component :).
Download and install https://github.com/Minei3oat/com_test .
The component contains a single view with five slides. The style of the component will show `accordion-toggle`s with class `collapsed` green and without red. This makes it easy to see the difference.
Now open a slide which is not open. The open slide will close, but its `accordion-toggle` don't gets the class `collapsed` as it gets when you close the slide yourself.
After applying this patch the class `collapsed` will get asseigned.
### Documentation Changes Required

none
